.github/workflows/ci.yml

```diff
@@ -12,11 +12,9 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - '2.6'
           - '2.7'
+          - '3.0'
```

Comment:
Worth adding 3.1 while we're here?

Reply:
I was going to, but Rubocop failed with [this issue](https://github.com/rubocop/rubocop/issues/10258). We need to bump Rubocop, which causes new cops to fail. I'd prefer to do that in a separate PR.